react/Gallery.js

```diff
@@ -99,30 +72,12 @@ Gallery.propTypes = {
   products: PropTypes.arrayOf(productShape),
   /** ProductSummary props. */
   summary: PropTypes.any,
-  /** Gap between items */
-  gap: PropTypes.oneOf(getGapPaddingValues()),
   /** Max Items per Row */
   maxItemsPerRow: PropTypes.number,
   /** Layout mode of the gallery in mobile view */
   mobileLayoutMode: PropTypes.oneOf(pluck('value', LAYOUT_MODE)),
   /** Min Item Width. */
   minItemWidth: PropTypes.number,
-  /** Render runtime mobile hint */
-  runtime: PropTypes.shape({
-    hints: PropTypes.shape({
-      mobile: PropTypes.bool.isRequired,
-    }).isRequired,
-  }).isRequired,
-}
-
-Gallery.defaultProps = {
-  products: [],
-  maxItemsPerRow: 5,
-  minItemWidth: 240,
-  mobileLayoutMode: LAYOUT_MODE[0].value,
 }
 
-export default compose(
-  withResizeDetector,
-  withRuntimeContext
-)(Gallery)
+export default withResizeDetector(Gallery)
```

Comment:
@matheusps did a hook to do this instead of use this HOC. I think you can use it too.

Reply:
where is this hoc?

Reply:
This hooks is @ `telemarketing` and is called `useMedia()`. (These custom hooks should be in a separate repo).